templates/cases/email/case_export.html

```diff
@@ -1,13 +0,0 @@
-{% load tz %}
```

Comment:
All of these export type specific templates are replaced by _utils/email/export.html_ and _utils/email/export.txt_